airflow/providers/databricks/hooks/databricks.py

```diff
@@ -388,6 +437,32 @@ def repair_run(self, json: dict) -> None:
         """
         self._do_api_call(REPAIR_RUN_ENDPOINT, json)
 
+    def get_cluster_state(self, cluster_id: str) -> ClusterState:
+        """
+        Retrieves run state of the cluster.
+
+        :param cluster_id: id of the cluster
+        :return: state of the cluster
+        """
+        json = {"cluster_id": cluster_id}
+        response = self._do_api_call(GET_CLUSTER_ENDPOINT, json)
+        state = response["state"]
+        state_message = response["state_message"]
+        return ClusterState(state, state_message)
```

Comment:
what about creating it via `from_json`?

Reply:
Nope, because the `from_json()` method converts json string value to `ClusterState` instance. Also,`response` is kind of python object type, so it is not suitable to use.